pkg/cmd/output.go

```diff
@@ -28,12 +28,159 @@ const (
 	AssetTypeCol        = "Asset Type"
 	CPUCostCol          = "CPU Cost"
 	RAMCostCol          = "RAM Cost"
+
+	PredictColWorkload     = "Workload"
+	PredictColReqCPU       = "CPU"
+	PredictColReqMemory    = "Mem"
+	PredictColMoCoreHours  = "Mo. core-hrs"
+	PredictColMoGibHours   = "Mo. GiB-hrs"
+	PredictColCostCoreHr   = "Cost/core-hr"
+	PredictColCostGiBHr    = "Cost/GiB-hr"
+	PredictColMoCostCPU    = "CPU/mo"
+	PredictColMoCostMemory = "Mem/mo"
+	PredictColMoCostTotal  = "Total/mo"
 )
 
 func formatFloat(f float64) string {
 	return fmt.Sprintf("%.6f", f)
 }
 
+func writePredictionTable(out io.Writer, rowData []predictRowData, currencyCode string, showCostPerResourceHr bool) {
+	t := makePredictionTable(rowData, currencyCode, showCostPerResourceHr)
+	t.SetOutputMirror(out)
+	t.Render()
+}
+
+func makePredictionTable(rowData []predictRowData, currencyCode string, showCostPerResourceHr bool) table.Writer {
+	t := table.NewWriter()
+
+	columnConfigs := []table.ColumnConfig{
+		table.ColumnConfig{
```

Comment:
no need to state the type as table.ColumConfig for each of this values as it's redundant. But not a blocker.